app/controllers/admin/movies_controller.rb

```diff
@@ -13,7 +14,43 @@ def index
     end
   end
 
+  def edit; end
+
+  def update
+    Movie.transaction do
+      @movie.update_attributes! movie_params
+    end
+    flash[:success] = t "update_success.movie"
+    redirect_to_condition = params[:button] == Settings.continue
+    redirect_to edit_admin_movie_path @movie if redirect_to_condition
+    redirect_to admin_movies_path unless redirect_to_condition
+  rescue ActiveRecord::RecordNotSaved
+    flash.now[:danger] = t :save_error
+    render :edit
+  end
+
+  def load_subtitles
+    @subtitles = Subtitle.where episode_id: params[:episode]
```

Comment:
`.where episode_id: params[:episode]` viết scope nhé

mà method này private chứ nhỉ

Reply:
Phương thức này dùng để khi người dùng chọn tập phim thì load phụ đề cho tập đó, `member{post :subtitles, to: "movies#load_subtitles"}`